pages/glossary/highest-possible-contrast.md

```diff
@@ -5,8 +5,6 @@ unambiguous: true
 objective: true
 ---
 
-The highest value of the [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) between two sets of colors (A and B).
-To calculate the highest value, find the colors in each set with the largest and smallest 
-[relative luminance](https://www.w3.org/TR/WCAG21/#dfn-relative-luminance) (`A-lum-max`, `A-lum-min`, `B-lum-max`, `B-lum-min`). The
-highest possible contrast is then the larger of the [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio)
-of `A-lum-max` with `B-lum-min` versus `A-lum-min` with `B-lum-max`.
+The highest value of the [contrast ratios][contrast ratio] between two sets of colors (A and B). That is, the highest [contrast ratio][] between one color of A and one color of B.
```

Comment:
I'm not keen on this phrase:

> That is, the highest [contrast ratio][] between one color of A and one color of B.

Which "one color" is this? How about a middle-ground solution:

```suggestion
The highest value of the [contrast ratios][contrast ratio] between two sets of colors (A and B). That is, the highest [contrast ratio][] between either the darkest color in A and the brightest color in B, or the brightest color in A and the darkest color in B.
```